salt/states/grafana4_user.py

```diff
@@ -53,10 +54,12 @@ def __virtual__():
 
 def present(name,
             password,
-            email,
+            email=None,
```

Comment:
It looks like you're making this optional but I don't see this change reflected in the docstring below.

Reply:
fixed in bbczeuz/salt@5f63c595bba1ebe61bd26f73024ed370037509b5